alloydb-jdbc-connector/pom.xml

```diff
@@ -80,11 +80,6 @@
       <artifactId>proto-google-cloud-alloydb-connectors-v1</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.api.grpc</groupId>
-      <artifactId>grpc-google-cloud-alloydb-v1alpha</artifactId>
```

Comment:
We need this dependency for the tests. Could you add the `<scope>test</scope>` instead of removing it? Does it solve your problem?

Reply:
I can - I fixed my problem (was an issue with the Spring Google BOM downgrading). This was just something I noticed and wanted to clean up. Sorry for the confusion.

I think this might help highlight the duplication in particular - you can see the same compile time dependency is defined twice.

![image](https://github.com/GoogleCloudPlatform/alloydb-java-connector/assets/2126119/01870ce7-9a45-44de-b6ec-5938e0efa73e)

Reply:
Bah! I didn't read hard enough, sorry about that - I see now they are indeed different dependencies. Closing. Thanks for the help!

Reply:
No problem.